extensions/frontend-tiles/src/integration-test/IntegrationTest.test.ts

```diff
@@ -0,0 +1,108 @@
+import { IModelApp } from "@itwin/core-frontend";
+import { expect } from "chai";
+import { TestContext } from "./TestContext";
+import { GraphicRepresentationStatus } from "../GraphicsProvider/GraphicRepresentationProvider";
+
+async function getExport(exportId: string, accessToken: string, urlPrefix?: string): Promise<any> {
+  const headers = {
+    /* eslint-disable-next-line @typescript-eslint/naming-convention */
+    Authorization: accessToken,
+    /* eslint-disable-next-line @typescript-eslint/naming-convention */
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
+  };
+
+  // obtain export for specified export id
+  const url = `https://${urlPrefix}api.bentley.com/mesh-export/${exportId}`;
+  try {
+    const response = await fetch(url, { headers });
+    const result = await response.json();
+    return result;
+  } catch (err) {
+    return undefined;
+  }
+}
+
+describe("MeshExportServiceIntegrationTest", () => {
+  let accessToken: any;
+  let urlPrefix: any;
+  let exportId: any;
+
+  before(async () => {
+
+    // obtain access token
+    const testContext = await TestContext.instance();
+    accessToken = testContext.getAccessToken();
+
+    const iModelId = process.env.MES_IMODEL_ID || "";
+    urlPrefix = process.env.imjs_url_prefix || "";
+
+    await IModelApp.startup();
+    const requestOptions: RequestInit = {
+      method: "POST",
+      headers: {
+        /* eslint-disable-next-line @typescript-eslint/naming-convention */
+        "Authorization": accessToken,
+        /* eslint-disable-next-line @typescript-eslint/naming-convention */
+        "Accept": "application/vnd.bentley.itwin-platform.v1+json",
+        /* eslint-disable-next-line @typescript-eslint/naming-convention */
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        iModelId,
+        changesetId: "",
+        exportType:"IMODEL",
+      }),
+    };
+
+    // initiate mesh export
+    const response = await fetch(`https://qa-api.bentley.com/mesh-export/`, requestOptions);
+    const result = JSON.parse(JSON.stringify(await response.json()));
+    exportId = result.export.id;
+  });
+
+  after(async () => IModelApp.shutdown());
+
+  it ("should obtain a tileset with a tile version supported by the frontend", async () => {
+    let exportComplete = false;
+    let tilesetUrl: any;
+
+    // poll for export status
+    const start = Date.now();
+    while (Date.now() - start < 90000) {
```

Comment:
what is this magical/mysterious number?

Reply:
This was originally what we were using as a timeout for the test, it's now been changed to five minutes on the suggestion of @arvindsvenkat, and it has been moved to the certa.json file. This while loop now doesn't include any magical numbers and only runs based on the status of the export.